extensions/interactions/Continue/directives/continue-interaction.directive.html

```diff
@@ -1,4 +1,13 @@
-<md-button ng-if="$ctrl.isInEditorMode" class="oppia-learner-confirm-button">
-  <[$ctrl.buttonText]>
-  <i class="fas fa-arrow-right" style="font-size: 19px; padding-top: 1.5px;"></i>
-</md-button>
+<div class="oppia-interactive-continue-interaction">
```

Comment:
Instead of creating this extra div should we use `oppia-interactive-continue-interaction` tag like we have done in line `extensions/interactions/EndExploration/directives/end-exploration-interaction.directive.html` (for consistency and less code!)

Reply:
We can't do that because the use of tags to apply styles is not possible in angular 8, therefore we have to keep the code under a div to use the angular 1 type tag.